python/Ganga/Utility/execute.py

```diff
@@ -194,7 +195,7 @@ def execute(command,
         python_setup (str): A python command to be executed beore the main command is
         eval_includes (str): An string used to construct an environment which, if passed, is used to eval the stdout into a python object
         update_env (bool): Should we update the env being passed to what the env was after the command finished running
-        return_code (int): This is the returned code from the command which is executed
+        return_code (list): This has element 0 modified to be the returned code from the command which is executed
```

Comment:
If this is going down this route, I'll revert the changes here and add a fixme message to the DiracUtils as this is starting to get into the scope of some invasive changes which could cause more problems than this PR solves.
I'll revert the changes to `execute.py` and update the GangaDirac appropriately.